### DISCUSSION TITLE
Nouveau mini event : devinez le poème dont sont tirés ces vers

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1579] [Nouvelles idées]
Lors d'une de vos aventures, vous serez amenés à croiser Pérotin, éminent ménestrel connu dans tout le royaume. Pour tester votre culture personnelle, celui-ci va vous proposer un bout d'un poème et vous devrez deviner parmi les X propositions données de quel poème provient les vers donnés. Si vous répondez juste, vous serez récompensés par une petite avancée de temps (la durée d'une bonne musique, somme toute). Cependant, si vous répondez faux, vous serez pénalisés d'une petite retenue du ménestrel, afin qu'il vous montre la différence entre les deux poèmes, afin de vous prouvez vos tords. (disons deux bonnes musiques)

En moins formel, le but serait de trouver à quel poème parmi un faible choix de proposition un vers appartient. Les poèmes pourraient soit être inventés (donc le mini event serait de plus en plus profitable à force d'apprendre les vers et les poèmes) soit ce serait des vraies poèmes connus (mais dans ce cas, il faudrait une API qui nous fournisse les informations). Pour ce qui est des bonus malus, je dirais +5min/-10min, donc équivalent en terme de gain perte à la lotterie, juste que la on peut pas modifier ses chances.

**Comment by BastLast** [comment]
faut check si ya une API pour ça déjà pour avoir des vers de poems randoms

**Reply by BastLast** [reply]
qui soient assez variés et qui restent quand même assez connus

**Comment by Thesmokingllama** [comment]
L'introduire en remplacement du mini event des gobelets ça pourrait être pas mal
